galaxy_ng/tests/performance/constants.py

```diff
@@ -0,0 +1,16 @@
+URLS = {
+    '/api/_ui/v1/settings/' : 0.5,
+    '/api/ui/v1/namespaces/' : 0.5,
+    '/api/_ui/v1/auth/login/' : 0.5,
+    '/api/_ui/v1/collection-versions/' : 0.5,
+    '/api/_ui/v1/distributions/' : 0.5,
+    '/api/_ui/v1/imports/collections/' : 0.5,
+    '/api/_ui/v1/landing-page/': 0.5,
+    '/api/_ui/v1/me/' : 0.5,
+    '/api/_ui/v1/remotes/' : 0.5,
+    '/api/_ui/v1/search/' : 0.5,
+    '/api/_ui/v1/settings/' : 0.5,
+    '/api/_ui/v1/synclists/': 0.5,
+    '/api/_ui/v1/users/': 0.5,
+    '/api/_ui/v1/tags/roles/': 0.5,
+}
```

Comment:
Maybe this object could get more nesting data, so we can play with different methods.

```py
URLS = {
    '/api/_ui/v1/settings/': {
         'baseline': 0.5,
         'method': 'GET'
    },
    '/api/other_url': {
        'baseline': 0.7,
        'method': 'POST',
        'payload': {...}
    }
}
```

Reply:
Thank you! I will integrate your advice.